src/CString.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace SDL2
+{
+	public struct CString
```

Comment:
Not totally thrilled with this class... not only is it a whole structure just for what's essentially a pointer, but it also ignores the UTF8 marshaler we've got.